packages/mobile/src/web3/saga.ts

```diff
@@ -203,6 +204,9 @@ function getPrivateKeyFilePath(account: string): string {
 
 function ensureAddressAndKeyMatch(address: string, privateKey: string) {
   const generatedAddress = getAccountAddressFromPrivateKey(privateKey)
+  if (!generatedAddress) {
+    throw new Error(`Failed to generate address from private key`)
+  }
```

Comment:
not sure how a `console.debug` slipped in below (215) but please convert it to Logger

Reply:
Done

Reply:
Removed.